windows-release/checkout.yml

```
@@ -19,3 +19,11 @@ steps:
 - script: git clone --progress -v --depth ${{ parameters.depth }} --branch $(Build.SourceBranchName) --single-branch $(Build.Repository.Uri) .
   displayName: 'git clone'
   condition: and(succeeded(), and(not(variables['GitRemote']), not(variables['SourceTag'])))
+
+- powershell: |
+    $checkout_commit = (git rev-parse HEAD)
+    if ($checkout_commit -ne '$(SourceCommit)') {
+        throw "Expected git commit '$(SourceCommit)' didn't match tagged commit '$(checkout_commit)'"
```

```suggestion
        throw "Expected git commit '$(SourceCommit)' didn't match tagged commit '$checkout_commit'"
```

This should be the last fix. I think we've got everything now

---

Fixed and pushed! :)